### PR TITLE
Store attributes like icons

### PR DESCRIPTION
**Pull request by natewiebe13**
Remove the namespaced constant and use in the same manner as icons

### CODE REVIEW

**Review by michaelsavchuk — APPROVED** [review]

**Comment by michaelsavchuk** [comment]
This looks good. Judging from the PR chain going on here, I think it's best to wait for #34 to be merged into this first before merging into `remove-javascript`?

**Comment by natewiebe13** [comment]
I was actually planning to do the opposite direction. They are against the other branches as they build on the previous. You'll see that Github automatically changed the base branch once the target branch was merged.

**Pull request merged**
